postcss.config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = ( { options: { preserveCssCustomProperties = true } } ) => {
+	const plugins = {
+		'postcss-custom-properties': {
+			importFrom: [
+				'./node_modules/@automattic/calypso-color-schemes/dist/calypso-color-schemes.css',
+			],
+			preserve: preserveCssCustomProperties,
+		},
+		autoprefixer: {},
+	};
+
+	return { plugins };
+};
```

Comment:
Would it be possible to re-use the one from `calypso-build`?

https://github.com/Automattic/wp-calypso/blob/ba0cadb01098d8352d84cd35c6a82ccc7c63de0c/packages/calypso-build/postcss.config.js

Reply:
I'm looking into that, thanks for the suggestion 👍

Reply:
It appears to be that calypso build postcss config relies on `require.resolve` which is a webpack feature and currently not supported by the process of building css for calypsoify - there we use gulp. Looking into if we can use `calypso-build` for calypsoify itself might be worth a shot though? (cc @ockham)

Reply:
Oh right so maybe this https://github.com/Automattic/jetpack/pull/12072 or https://github.com/Automattic/jetpack/pull/12127